index.html

```diff
@@ -959,8 +973,8 @@ <h6 class="after-cover-text subs-title mt-3 ml-3">SUBSCRIBE</h6>
                    </div>  
```

Comment:
Please add padding around these and align them correctly as present on the original page.